client/src/components/ReviewCard/ReviewCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { Card, CardHeader, Avatar, IconButton, CardContent, Typography, Menu, MenuItem } from '@material-ui/core';
```

Comment:
It would be cleaner if you broke this into multiple lines.

Reply:
I tried moving it but seems the linting is config is keeping it this way for now

Reply:
This is fine, as long as we are following the linter 😄